shade-test/jar-file/src/test/java/org/testcontainers/JarFileShadingTest.java

```diff
@@ -0,0 +1,79 @@
+package org.testcontainers;
+
+import org.assertj.core.api.ListAssert;
+import org.junit.*;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.*;
+
+import static java.util.Collections.emptyMap;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JarFileShadingTest {
+
+    private static FileSystem fileSystem;
+
+    private static Path root;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        Path path = Paths.get("..", "..", "core", "target", "testcontainers-0-SNAPSHOT.jar");
```

Comment:
In this test you rely on the fact that someone ran mvn install on the core project right? Once he has done that you can the test from the IDE.
I'm saying the same thing - someone must run mvn install on the core project *and* on the project it self. Then you run the test from the IDE - same. 
So both rely on manual mvn operation. 
Am I missing something here?